### PR TITLE
Refine the readme file

### PR DESCRIPTION
**Pull request by MaggieCabrera**
WIP

To do:

- How to contribute a new theme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,25 +4,27 @@ Welcome to the repository for Block Themes built by the WordPress community.
 
 ## About
 
+This repo is starting because WordPress is in a new era of theming and site creation overall, to encourage folks both experienced and new to WordPress to contribute more themes and demonstrate what can be done and how. This way, we learn from / inspire each other faster.
```
Comment (jffng) [inline README.md]: Just some wording suggestions. What do you think the benefit to submitting them here is versus the wordpress.org/themes repo directly?

```suggestion
WordPress is in a new era of theming and site creation overall. This repo exists to encourage folks both experienced and new to WordPress to contribute more themes — demonstrating what can be done and how. This way, we learn from and inspire each other faster.
```

Comment (MaggieCabrera) [reply README.md]: Good suggestion, I'll expand on the benefits of this idea!

File: README.md [hunk 2]
```diff
@@ -36,6 +36,12 @@ If it's your first time building a Block Theme, we suggest checking the Resource
 
 ### How to contribute a new theme
 
+You can contribute with code, designs or discussion. 
+
+If you want to submit a theme design, you can include a figma link or a zip file of your theme created using the Create Block Theme plugin. You can use the [Twenty Twenty-Three Figma Mockups](https://www.figma.com/community/file/1139275543113752375) as a reference.
+
+We will kickstart this innitiative building child themes of Twenty Twenty-Three. The themes can include as many or as few templates as you prefer, and the rest will be inherited by the parent theme.
```
Comment (mikachan) [inline README.md]:
```suggestion
We will kickstart this initiative by building child themes of Twenty Twenty-Three. The themes can include as many or as few templates as you prefer, and the rest will be inherited from the parent theme.
```

File: README.md [hunk 3]
```diff
@@ -4,25 +4,25 @@ Welcome to the repository for Block Themes built by the WordPress community.
 
 ## About
 
-## Contributing
+WordPress is in a new era of theming and site creation overall. This repo exists to encourage folks both experienced and new to WordPress to contribute more themes — demonstrating what can be done and how. This way, we learn from and inspire each other faster.
```
Comment (scruffian) [inline README.md]:
```suggestion
WordPress is in a new era of theming and site creation. This repo encourages both experienced and new  WordPress contributors to create more themes — demonstrating what can be done and how. This way we learn from and inspire each other.
```

File: README.md [hunk 4]
```diff
@@ -4,25 +4,25 @@ Welcome to the repository for Block Themes built by the WordPress community.
...
 
-You can contribute with code, designs or discussion.
+The aim of this repo is to submit the block themes to the WordPress.org themes directory under the WordPress user, ensuring the quality of the code and design of said themes. Hopefully this initiative will help more users without prior theming knowledge or who don't code to create high quality themes by themeselves using the new tools available to us.
```
Comment (scruffian) [inline README.md]:
```suggestion
The aim of this repo is to submit the block themes to the WordPress.org themes directory under the WordPress user, ensuring the quality of the code and design of said themes. This initiative will help more users without prior theming knowledge or who don't code to create high quality themes by themselves using the new site building tools.
```

File: README.md [hunk 5]
```diff
@@ -36,6 +36,12 @@ If it's your first time building a Block Theme, we suggest checking the Resource
 
 ### How to contribute a new theme
 
+You can contribute with code, designs or discussion. 
+
+If you want to submit a theme design, you can include a figma link or a zip file of your theme created using the Create Block Theme plugin. You can use the [Twenty Twenty-Three Figma Mockups](https://www.figma.com/community/file/1139275543113752375) as a reference.
```
Comment (scruffian) [inline README.md]:
```suggestion
To submit a theme design, include a Figma link or a zip file of your theme created using the Create Block Theme plugin. You can use the [Twenty Twenty-Three Figma Mockups](https://www.figma.com/community/file/1139275543113752375) as a reference.
```

File: README.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ Welcome to the repository for Block Themes built by the WordPress community.
 
 WordPress is in a new era of theming and site creation overall. This repo exists to encourage folks both experienced and new to WordPress to contribute more themes — demonstrating what can be done and how. This way, we learn from and inspire each other faster.
 
-The aim of this repo is to submit the block themes to the WordPress.org themes directory under the WordPress user, ensuring the quality of the code and design of said themes.
+The aim of this repo is to submit the block themes to the WordPress.org themes directory under the WordPress user, ensuring the quality of the code and design of said themes. Hopefully this initiative will help more users without prior theming knowledge or who don't code to create high quality themes by themeselves using the new tools available to us.
```
Comment (jffng) [inline README.md]:
```suggestion
The aim of this repo is to submit the block themes to the WordPress.org themes directory under the WordPress user, ensuring the quality of the code and design of said themes. Hopefully this initiative will help more users without prior theming knowledge or who don't code to create high quality themes by themselves using the new tools available to us.
```

**Review by jffng — APPROVED** [review]
LGTM! Let's get the party started.

**Pull request merged**